src/lib/validations/site-type.js

```diff
@@ -42,45 +39,22 @@ export const siteTypeValidations = {
 			}\n`
 		);
 
-		// if site is a multisite but import sql is not
-		if ( ! isMultiSite ) {
-			if ( isMultiSite && ! isMultiSiteSqlDump ) {
-				await track( 'import_sql_command_error', {
-					error_type: 'multisite-but-not-multisite-sql-dump',
-				} );
-				throw new Error(
-					'You have provided a non-multisite SQL dump file for import into a multisite.'
-				);
-			}
-
-			// if site is a single site but import sql is for a multi site
-			if ( ! isMultiSite && isMultiSiteSqlDump ) {
-				await track( 'import_sql_command_error', {
-					error_type: 'not-multisite-with-multisite-sql-dump',
-				} );
-				throw new Error(
-					'You have provided a multisite SQL dump file for import into a single site (non-multisite).'
-				);
-			}
-		} else {
-			// validations for when the intended import of for a multisite subsite
-			if ( ! isMultiSite ) {
-				await track( 'import_sql_command_error', {
-					error_type: 'not-multisite-with-subsite-sql-dump',
-				} );
-				throw new Error(
-					'You have requested a subsite SQL import into a non-multisite installation.'
-				);
-			}
+		if ( ! isMultiSite && isMultiSiteSqlDump ) {
+			await track( 'import_sql_command_error', {
+				error_type: 'not-multisite-with-multisite-sql-dump',
+			} );
+			throw new Error(
+				'You have provided a multisite SQL dump file for import into a single site (non-multisite).'
+			);
+		}
 
-			if ( ! isMultiSiteSqlDump ) {
-				await track( 'import_sql_command_error', {
-					error_type: 'subsite-import-without-subsite-sql-dump',
-				} );
-				throw new Error(
-					'You have requested a subsite SQL import but have not provided a subsite compatiable SQL dump.'
-				);
-			}
+		if ( isMultiSite && ! isMultiSiteSqlDump ) {
```

Comment:
Is there a reason we don't want to allow importing a sql file that just has site 1 data and / or just top-level `wp_*` tables?  Seems like there could be a few use cases for this.

Reply:
Actually, we need to beef up our validation here.  There are columns in the `wp_users` table for multisites that don't exist in single-site installations.  Thinking about it now, we might want to use that difference as the check for `isMultiSiteSqlDump` instead of if more than one site exists in the SQL.